app/controllers/bookings_controller.rb

```diff
@@ -1,8 +1,13 @@
 class BookingsController < ApplicationController
   before_action :authenticate_user!, except: [:index]
-  
-  def index
-    @bookings = Booking.all
```

Comment:
We need 2 isntance variabels to store two seperate collections of bookings:

1. `@requested_bookings = Booking.where(user: current_user.id)`
2. `@received_bookings = booking_array`